keps/sig-storage/596-csi-inline-volumes/README.md

```diff
@@ -377,11 +377,10 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - [x] Fix for [#89290 - CSI Inline volume panic when calling applyFSGroup](https://github.com/kubernetes/kubernetes/issues/89290)
 - [x] Fix for [#79980 - CSI volume reconstruction](https://github.com/kubernetes/kubernetes/issues/79980)
 - [x] Updated documentation as described in [Security Considerations](#security-considerations) and [Read-only volumes](#read-only-volumes)
-- [ ] Upgrade / downgrade manual testing, document results in the [upgrade / rollback section](#rollout-upgrade-and-rollback-planning).
-- [ ] Provide measurements for the [Scalability section](#scalability) (time taken to start a pod)
-- [ ] Ensure our sponsored [NFS](https://github.com/kubernetes-csi/csi-driver-nfs) and [SMB](https://github.com/kubernetes-csi/csi-driver-smb) CSI drivers align with the new guidance in [Security Considerations](#security-considerations)
-- [ ] Conformance tests implemented / promoted
```

Comment:
I think even though full end to end functionality may not be easily supported in conformance, we can still test API operations, similar to https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/csistoragecapacity.go

Reply:
Added as a separate commit in https://github.com/kubernetes/kubernetes/pull/111258